eisbuk-admin/src/store/selectors/slots.ts

```diff
@@ -69,6 +70,24 @@ const getSlotsForADay = (dayStr: string) => (state: LocalStore) => {
   return getSafe(() => state.firestore.data.slotsByDay![monthStr][dayStr]);
 };
 
+/**
+ * HOF that creates a selector for view-specific slots by view in a single record, keyed by (day) date
+ * @param view tab viewed by customer (ice, off-ice)
+ * @returns record of view-specific ice slots, keyed by day, grouped together (regardless of month)
+ */
+export const getSlotsByView = (view: CustomerRoute) => (
+  state: LocalStore
+): any =>
```

Comment:
Please, if you slap `any`'s to speed up the process, add a `@TODO` and fix later. We're purposely enforcing return type on functions rule.